pages/linux/losetup.md

```diff
@@ -17,3 +17,11 @@
 - Detach a given loop device:
 
 `sudo losetup -d /dev/{{loop}}`
+
+- Attach a file to a new free loop device and scan the device for partitions:
```

Comment:
Please add these examples before the detach ones.

Reply:
@z7y8hsBP not sure I see how this is resolved, unless you opened a new PR

Reply:
Hmm, I added and commited it in my fork, should automatically add it to the PR. But I can open a new PR.

Reply:
https://github.com/z7y8hsBP/tldr/commit/c8ac8c6f88e552b7e342fe167d8b6ce61ab32d3c#diff-39cc51be6e08db5cd9c9816ce9003387

Reply:
Ah, I see @z7y8hsBP. Perhaps it's because this PR has already been merged. Thanks for opening a new one :-)